src/data_provider/tests/sysInfoNetworkWindows/sysInfoNetworkWindows_test.cpp

```diff
@@ -28,7 +28,7 @@ class SysInfoNetworkWindowsWrapperMock: public INetworkInterfaceWrapper
     virtual ~SysInfoNetworkWindowsWrapperMock() = default;
     MOCK_METHOD(int, family, (), (const override));
     MOCK_METHOD(std::string, name, (), (const override));
-    MOCK_METHOD(std::string, adapter, (), (const override));
+    MOCK_METHOD(std::string, description, (), (const override));    
```

Comment:
In `inetworkWrapper.h` interface, there is no field called "description". This has been changed by "adapter" in previous PR changes.